forms-flow-api/src/formsflow_api/resources/form_embed.py

```diff
@@ -121,28 +163,16 @@ def post():
         }
         ```
         """
-        formio_url = current_app.config.get("FORMIO_URL")
-        web_url = current_app.config.get("WEB_BASE_URL")
-        data = request.get_json()
-        try:
-            (
-                response,
-                status,
-            ) = CombineFormAndApplicationCreate.application_create_with_submission(
-                data, formio_url, web_url, request.headers["Authorization"]
-            )
-            return response, status
-        except BusinessException as err:
-            return err.error, err.status_code
+        return EmbeddCommonMethods.post()
 
 
 @cors_preflight("GET,OPTIONS")
-@API.route("/form/<string:path>", methods=["GET", "OPTIONS"])
-class Form(Resource):
+@API.route("/internal/form/<string:path>", methods=["GET", "OPTIONS"])
```

Comment:
internal ?

Reply:
2 endpoints for form embedding `/internal` for keycloak authentication `/external` for custom authentication